tests/evaluate/test_default_evaluator.py

```diff
@@ -2208,3 +2210,17 @@ def test_eval_results_table_json_can_be_prefixed_with_metric_prefix():
     client = mlflow.MlflowClient()
     artifacts = [a.path for a in client.list_artifacts(run.info.run_id)]
     assert f"{metric_prefix}eval_results_table.json" in artifacts
+
+
+def test_extract_raw_model_for_sklearn():
+    mlflow_model = Model()
+    model_impl = Pipeline([("predict", LogisticRegression())])
+    mlflow.pyfunc.add_to_model(mlflow_model, loader_module="mlflow.sklearn")
+    pyfunc_model = mlflow.pyfunc.PyFuncModel(model_meta=mlflow_model, model_impl=model_impl)
+    _, raw_model = _extract_raw_model(pyfunc_model)
+    assert raw_model == model_impl
+
+    model_impl = _SklearnModelWrapper(model_impl)
+    pyfunc_model = mlflow.pyfunc.PyFuncModel(model_meta=mlflow_model, model_impl=model_impl)
+    _, raw_model = _extract_raw_model(pyfunc_model)
+    assert raw_model == model_impl
```

Comment:
Instead of testing, can we test `mlflow.evaluate`?

Reply:
Just because `_extract_raw_model` works fine doesn't mean it works fine in `mlflow.evaluate.`

Reply:
In general, we should test a public API, instead of implementatio details like `_extract_raw_model`.